_resourceexample/giffgaff.md

```diff
@@ -0,0 +1,12 @@
+---
+title: giffgaff's Design System
+link: https://www.giffgaff.design/
+image: https://www.giffgaff.design/og-image.png
```

Comment:
@dbenmore any way you could provide this image locally to live amongst the other images? Just trying to make sure things don't get broken as time goes by and resources shift around.

Reply:
@bradfrost done - please let me know if it's ok?